ibis/file/csv.py

```diff
@@ -1,11 +1,47 @@
+import numpy as np
 import pandas as pd
+
+import ibis.expr.datatypes as dt
 import ibis.expr.operations as ops
+
 from ibis.file.client import FileClient
+
+from ibis.pandas.api import PandasDialect
 from ibis.pandas.core import pre_execute, execute  # noqa
 from ibis.pandas.client import pandas_dtypes_to_ibis_schema
 from ibis.pandas.execution.selection import physical_tables
 
 
+_IBIS_TO_PANDAS_DTYPE = {
```

Comment:
@kszucs Does a function for this exist already? I think you might've written one in the PR where you introduced `infer`.

Reply:
I'm currently refactoring this one.